packages/swc-configs/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { Config } from "jest";
+import { config as swcConfig } from "./swc.jest.ts";
```

Comment:
if type:module has been removed, is there a reason why we specify extensions?

Reply:
It is still an ESM package.